packages/postgres-database/src/simcore_postgres_database/models/workspaces_access_rights.py

```diff
@@ -0,0 +1,60 @@
+import sqlalchemy as sa
+from sqlalchemy.sql import expression
+
+from ._common import column_created_datetime, column_modified_datetime
+from .base import metadata
+from .groups import groups
+from .workspaces import workspaces
+
+workspaces_access_rights = sa.Table(
+    "workspaces_access_rights",
+    metadata,
+    sa.Column(
+        "workspace_id",
+        sa.BigInteger,
+        sa.ForeignKey(
+            workspaces.c.workspace_id,
+            name="fk_workspaces_access_rights_id_workspaces",
+            onupdate="CASCADE",
+            ondelete="CASCADE",
+        ),
+        doc="Workspace unique ID",
+    ),
+    sa.Column(
+        "gid",
```

Comment:
MINOR: again, sometimes we use `group_id` others `gid`.

Reply:
Yes, generally we are not consistent. Here I am consistent with tables such as wallet_to_groups, projects_to_groups, ... I don't see it as a huge issue, but of course, any time we agree on a convention we should consolidate in separate PR.